Makefile

```diff
@@ -1,8 +1,10 @@
 UNAME := $(shell uname)
 
+transifex_langs = "ar,fr,es_419,zh_CN"
```

Comment:
Do we want to have these defaults here or in atlas? If we want the defaults in atlas then we could move this line down inside the `ifeq ($(OPENEDX_ATLAS_PULL),)` block

Reply:
In both actually. I have two questions in the description about language json files that I could use some answers for:
 
  - https://github.com/openedx/studio-frontend/pull/380#issue-1710468372

Reply:
It looks like those questions have been removed from that comment in the most recent edit, are you still looking for clarification on them?

Reply:
I've removed the questions because I ended up finding the method on how to load the `.json` on Open edX and added the note in the code: https://github.com/openedx/studio-frontend/pull/380/files#diff-c52f08377642c0ca5942d104abb4f4b9acda95dc35455c9785e4a62fbda130e9R55-R64